tests/README.md

```diff
@@ -12,29 +11,42 @@ There are two types of tests:
 
 1. Performing an action: To test an action that is called by the backend for
    write requests.
-   
-   * `action`: Name of the action. See backend health output for all action names.
-   
+
+   * `action`: Name of the action. See backend health output for all action
+     names.
+
    * `is_allowed`: Determines the expected result (`true` or `false`).
 
    * `payload`: User-Payload that is used for the action.
 
 2. Reading FQFields: To test whether the user can see FQFields or not.
 
-   * `fqfields`: A list of fields to be tested.
-   
-   * `can_see`: A list of fields that the user is expected to see. Omit fields that the user can not see.
-   
+  * `fqfields`: A list of fields to be tested.
+
+  * `fqids`: List of fqids. Adds all fields of the objects to the fqfields
+    attribute.
+
+  * `can_see`: A list of fields that the user is expected to see. This field is
+    the expected exact outout of the `RestrictFQFields` function. A value can also
+    be a fqid. In this case, all fields of the collection should be seen.
+
+  * `can_see_at_least`: Like `can_see` but does not throw an error, if more
+    fields are returned.
+
+  * `can_not_see`: A list of fields that the user is now allowed to see. A value
```

Comment:
Minor proposal: Add `can_not_see_at_least`

Reply:
Other proposal:

can_see means "can see at least"
can_not_see means "can not see at least"
can_see_only or can_see_exact is the current can_see and the exact output of the restrict interface.